.github/ISSUE_TEMPLATE/issue_template.md

```diff
@@ -0,0 +1,13 @@
+## Expected Behavior
+
+##Actual Behavior
+
+##Steps to Reproduce the Problem
+1.
+2.
+3.
+
+##Specifications
+- Version:
+- Platform:
+- Subsystem:
```

Comment:
I think that will be great to describe in comments what user need to write in each section.

This looks some like that

```
## Long story short

<!-- Please describe your problem and why the fix is important. -->
```